test/jb_controller/supports_interface.test.js

```diff
@@ -0,0 +1,93 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+import interfaceSignatures from '../helpers/interface_signatures.json';
+
+import jbDirectory from '../../artifacts/contracts/JBDirectory.sol/JBDirectory.json';
```

Comment:
extreme nit> We tried (maybe some escaped tho) keeping alphabetisation where possible

Reply:
Good point, wasn't aware of that, I made some changes to alphabetise but I might have forgotten some things. I'll have to get used to it